code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -93,6 +94,8 @@
 
 /obj/item/weapon/card/id/examine(mob/user)
 	..()
+	if(registered_race && registered_race.id != "human")

```

Comment:
remove the "not human" part, there's no reason to hide that it's a human id, if anything it adds to the specisim RP by being able to brag about the human tag on your ID.